docs/packages.md

```diff
@@ -21,18 +21,18 @@ Several packages are registered with OpenWhisk. You can get a list of packages i
   $ wsk package list /whisk.system
   ```
   ```
-  packages
-  /whisk.system/cloudant                                                 shared
-  /whisk.system/alarms                                                   shared
-  /whisk.system/watson                                                   shared
-  /whisk.system/websocket                                                shared
-  /whisk.system/weather                                                  shared
-  /whisk.system/system                                                   shared
-  /whisk.system/utils                                                    shared
-  /whisk.system/slack                                                    shared
-  /whisk.system/samples                                                  shared
-  /whisk.system/github                                                   shared
-  /whisk.system/pushnotifications                                        shared
+  packages		
+  /whisk.system/alarms													 shared
+  /whisk.system/cloudant   												 shared
+  /whisk.system/github 													 shared
+  /whisk.system/pushnotifications										 shared
```

Comment:
Shared is not aligned properly in the doc when viewing it for pushnotifications and samples.